assembl/static2/js/app/client.js

```diff
@@ -40,16 +46,28 @@ const networkInterface = createNetworkInterface({
 networkInterface.use([
   {
     applyMiddleware: function (req, next) {
-      Sentry.addBreadcrumb({
-        category: 'graphql',
-        message: `GraphQL operation: ${req.request.operationName}`,
-        data: {
-          variables: req.request.variables
-        },
-        level: 'info'
-      });
+      if (!req.options.headers) {
+        req.options.headers = {}; // Create the header object if needed.
+      }
+      co(async () => {
+        if (useCSRFProtection === 'true') {
+          const responseToken = await getCSRFToken();
+          if (responseToken.text === 'ok') {
+            req.options.headers['X-XSRF-TOKEN'] = Cookies.get('_csrf');
```

Comment:
Maybe `X-CSRF-TOKEN` name instead? No preference though...

Reply:
Ripped this from elsewhere wither CSRF or XSRF seems to be the standard - you want me to change before commit ?

Reply:
Nah, if you're happy with this option, we can keep it. I have no hard ops on this.